help/destinations/catalog/advertising/liveramp-onboarding.md

```diff
@@ -64,6 +64,9 @@ To authenticate to the destination, fill in the required fields and select **[!U
 
 ![Sample screenshot showing how to authenticate to the destination using SFTP with password](../../assets/catalog/advertising/liveramp-onboarding/liveramp-sftp-password.png)
 
+* **[!UICONTROL Port]**: The port used for your [!DNL LiveRamp - Onboarding] storage location.  This is contingent on your geographic region.
+  * US: Use Port "22"
+  * APAC: Use Port "2222"     
```

Comment:
Is APAC supported too?  The destination config that you submitted uses NA, AU, and EU. I don't see APAC there.

Reply:
I've edited to reflect AU rather than APAC to avoid any confusion.